### DISCUSSION TITLE
AI科技改變運動競賽

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2038] [General]
https://happylee.blog/learn/ai-judgement-in-olympics/

大多數運動中都有些許主觀性的介入，而這些判斷可能會對比賽結果產生重大影響。

最近奧運比賽開始，從小練習跆拳道的我非常期待看到比賽，然而跆拳道中「品勢」這個項目由於是由裁判評分（主觀性很大），導致遲遲無法進入奧運。近期，因AI的發展，許多運動的判罰也變得客觀許多，例如：最為人熟悉該是鷹眼以及AI評分輔助系統等等......這也讓由裁判給分的運動更加公正。ex:體操、跳水

相信人工智慧的進步，能夠為許多運動帶來更公平的比賽。
你認為AI還能在體育圈如何運用？歡迎分享

**Comment by Clare0808** [comment]
感覺AI對於各種運動都有很大的益處。之前有看過有選手利用AI去分析自己的體能、身體素質等等的條件，透過AI給的建議努力的精進自己，最後成功得到比賽的冠軍。只要適當的運用AI，對於我們生活各處都是利益。

**Comment by peresonacifer** [comment]
說到ai參與體育競技，我之前有看到一部關於足球選手朝ai機器人射門的片段，我印象中ai守門員可說是防得天衣無縫，那畫面彷彿是明星球員在挑戰頂級守門員，而當他們成功時，那快樂一定是難以體會的，畢竟ai機器人一定可以比人類更優秀，而我們可以把它們當作練習對象來精進自我，因此將來的運動賽事肯定會更精采。

**Comment by Penny-712** [comment]
像是法網這種很盛大的賽事到現在都還沒有鷹眼系統的判決，因為紅土的場地會有網球落到地上的痕跡，但就常常看到主審及場上球員意見不一致的情況產生，雖然說這也是比賽的一部分，也是這項賽事的傳統，不過我個人覺得用鷹眼判決還是比較公平。(像是之前女單好手Coco Gauff就因為不滿主審的判決而淚灑球場)

**Comment by ds0w0** [comment]
雖然我不常觀看體育賽事，但我認為AI能在體育圈發揮更大的作用。除了你提到的輔助裁判，AI還能在賽事戰術分析方面提供即時數據，幫助教練和選手做出更精準的判斷。此外，AI可以預測運動員的受傷風險並提供專屬的訓練計劃，進一步保護運動員的健康。這些應用能夠使比賽更加公平和客觀，也有助於提升整體運動表現。

**Comment by hunteryang0108** [comment]
在體育賽事中導入AI技術確實是一個很好的想法，AI的出現能大大降低人類的負擔，相較於人類來評分，AI評分也比較不容易造成爭議。我相信未來會有越來越多的領域融入AI技術，讓人類的發展更加迅速。

**Comment by CtyceT** [comment]
我認為AI作為工具輔助體育選手就非常實用了，例如分析選手的能力和成績來制定專屬於他的訓練計畫，或是在賽場上預測其他選手的能力來預先應對等。只要AI能夠普及使用，那麼運動競賽會更加的有趣與公平。
